democracylab/emails.py

```diff
@@ -222,20 +223,19 @@ def send_volunteer_application_email(volunteer_relation, is_reminder=False):
     email_template = HtmlEmailTemplate()\
         .subheader("Opportunity Information:")\
         .text_line("Title: {role}".format(role=role_details.display_name))\
-        .text_line("Organization: {project}".format(project=project.project_name))\
-        .text_line("Date: {currentdate}".format(currentdate=models.DateTimeField(auto_now_add=True, null=True)))\
+        .text_line("Organization: {projectname}".format(projectname=project.project_name))\
+        .text_line("Date: {currentdate}".format(currentdate=datetime_to_string(timezone.now(), DateTimeFormats.DATE_LOCALIZED)))\
         .subheader("Volunteer Information:")\
         .text_line("Name: {firstname} {lastname}".format(
             firstname=user.first_name,
             lastname=user.last_name))\
-        .text_line("Email: {email}".format(email=Html.a(href='mailto:' + user.email, text=user.email)))\
+        .text_line("Email: {email}".format(email=Html.a(href='mailto:{useremail}'.format(useremail=user.email), text=user.email)))\
```

Comment:
The href output here is missing quotes, per my dev console. That is, I get this:
`<a href=mailto:email@address.com>email@address.com</a>`

instead of what I want, which is this:
`<a href="mailto:email@address.com">email@address.com</a>`

Also it feels like nested .format() is pretty messy, is there a better way to do it?

Reply:
There's an issue with the Html.a helper.  It's not including quotes for the href part.

Also, for simple string appends like this you can use addition instead.
```suggestion
        .text_line("Email: " + Html.a(href='mailto:' + user.email, text=user.email))\
```

Reply:
Made that change, and I think I made Html.a include quotes the way we want on top. Triple quotes apparently do it?